ccd-definition/ComplexTypes/CareSupervision/InterimEndDate.json

```diff
@@ -16,5 +16,14 @@
     "ElementLabel": " ",
     "SecurityClassification": "Public",
     "FieldShowCondition": "type = \"NAMED_DATE\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "InterimEndDate",
+    "ListElementCode": "endDateTime",
+    "FieldType": "DateTime",
+    "ElementLabel": "Use 24 hour clock, for example 15 30 00",
```

Comment:
Should this not be hint text instead of the label?

Reply:
OH, i've changed :) was originally using eventHintText which didn't work - changed it